homeassistant/components/overkiz/water_heater.py

```diff
@@ -0,0 +1,28 @@
+"""Support for Overkiz water heater devices."""
```

Comment:
You need to exclude this from codecoverage in `.coveragerc`, or you would need to write tests.